lib/Screens/AuthRegistration/support_widgets.dart

```diff
@@ -30,46 +32,63 @@ class SignInTab extends StatelessWidget {
           style: TextStyle(color: Colors.white, fontSize: 15),
         ),
         TextButton(
-            onPressed: () {},
-            child: Text(
-              'Sign In',
-              style: TextStyle(
-                fontWeight: FontWeight.bold,
-                fontSize: 16,
-                color: Colors.white,
-              ),
-            ))
+          onPressed: () {
+            keyRegisterationButton.currentState!.changeButtonText('Boo');
```

Comment:
didn't worked with referencing on the other widget states, TBH. And I'm not sure if it is legal haha.
But OK, if it crying about NULL, wrap it inside of NULL check, like this 
if (keyRegisterationButton.currentState != null) {
  keyRegisterationButton.currentState!.changeButtonText('Boo');
}

Reply:
Yes,  `keyRegisterationButton.currentState` is null. That's why the code is not working. But I don't know why it is null when I initialized above like this. `GlobalKey<RegistrationButtonState> keyRegisterationButton = GlobalKey();` :(